apps/voting/contracts/Voting.sol

```diff
@@ -58,14 +63,20 @@ contract Voting is IForwarder, AragonApp {
     mapping (uint256 => Vote) internal votes;
     uint256 public votesLength;
 
+    uint64 public overruleWindow;
+    mapping (address => address) internal representatives;
+
     event StartVote(uint256 indexed voteId, address indexed creator, string metadata);
     event CastVote(uint256 indexed voteId, address indexed voter, bool supports, uint256 stake);
+    event ProxyVote(uint256 indexed voteId, address indexed voter, address indexed representative, bool supports, bool succeed);
```

Comment:
Note that both events will be emitted actually, I added this one to avoid changing the signature of `CastVote`. IMO, logging only the values that will allow us to match them should be enough